reana_workflow_engine_serial/utils.py

```diff
@@ -57,9 +36,7 @@ def build_job_spec(
     job_spec = {
         "image": image,
         "compute_backend": compute_backend,
-        "cmd": 'bash -c "cd {0} ; {1} "'.format(
-            workflow_workspace, escape_shell_arg(command)
-        ),
+        "cmd": "cd {0} && {1}".format(workflow_workspace, command),
```

Comment:
Background why the `cd {workflow_workspace}` is kept https://github.com/reanahub/reana-job-controller/pull/305#pullrequestreview-613367088.